esmvaltool/preprocessor/_io.py

```diff
@@ -51,11 +51,28 @@ def concatenate_callback(raw_cube, field, _):
                 coord.units = units
 
 
+def _adjust_cube_attributes(cubes):
+    """
+    Fixes attribute mismatch in cubes causing problems during
+    `concatenate_cube` operations.
+    """
+    from iris.experimental.equalise_cubes import equalise_attributes
+
+    equalise_attributes(cubes)
```

Comment:
This should no longer be needed since issue https://github.com/ESMValGroup/ESMValTool/issues/538 was fixed in PR #729.

Reply:
Yes, it is not needed. Removed by 6cc101b7b663d1574f466e078674a62835de34e5